wolfssl/test.h

```diff
@@ -2001,16 +2013,111 @@ static WC_INLINE int StackSizeCheck(func_args* args, thread_func tf)
     }
 
     free(myStack);
+#ifdef DEBUG_STACK_SIZE_VERBOSE
+    StackSizeCheck_myStack = 0;
+    return (int)((size_t)status);
+#endif
 
     used = stackSize - i;
     printf("stack used = %d\n", used);
 
     return (int)((size_t)status);
 }
 
+#ifdef DEBUG_STACK_SIZE_VERBOSE
```

Comment:
These additional stack debugging macros look handy. Can you add a comment with example usage? The ones  in tests.c appear to be mapped to a printf stub.

Reply:
I've fixed up this mechanism to be really-right, and it's now enabled with --enable-stacksize-verbose.